scripts/publish-docs.sh

```diff
@@ -48,6 +48,13 @@ function clean_up_gh_pages() {
   rm -rf gh-pages
 }
 
+# Enforce JDK8 to keep javadoc format consistent for all versions:
+java_version=$(java -version 2>&1 | grep 'version' 2>&1 | awk -F\" '{ split($2,a,"."); print a[1]"."a[2]}')
```

Comment:
As we use `gradle` to create javadocs, should we be considering if `org.gradle.java.home` is set?

Reply:
You are right, we need to extract the version of JVM that gradlew sees. I've updated the script to use `./gradlew -versions`.